app.test.js

```diff
@@ -29,22 +29,54 @@ describe("GET /api/topics", () => {
 });
 
 describe("GET /api/topics/:article_id", () => {
-  // test("Responds with correct article id", () => {
-  //   return request(app)
-  //     .get("/api/articles/1")
-  //     .expect(200)
-  //     .then((res) => {
-  //       const article = res.body.article_id;
-  //       expect(articles).toBeInstanceOf(Object);
-  //       expect(article).toMatchObject({
-  //         author: expect.any(String),
-  //         title: expect.any(String),
-  //         article_id: expect.any(Number),
-  //         topic: expect.any(String),
-  //         created_at: expect.any(String),
-  //         votes: expect.any(Number),
-  //         comment_count: expect.any(Number),
-  //       });
-  //     });
-  // });
+  test("Responds with correct article id object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.article
+        expect(article).toMatchObject({
+          author: "butter_bridge",
+          title: "Living in the shadow of a great man",
+          article_id: 1,
+          body: "I find this existence challenging",
+          topic: "mitch",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+        });
+      });
+  });
+  test("Responds with correct article id object", () => {
+    return request(app)
+      .get("/api/articles/10")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.article
+        expect(article).toMatchObject(      {
+          author: "rogersop",
+          title: "Seven inspirational thought leaders from Manchester UK",
+          article_id: 10,
+          body: "Who are we kidding, there is only one, and it's Mitch!",
+          topic: "mitch",
+          created_at: "2020-05-14T04:15:00.000Z",
+          votes: 0
+        });
+      });
+  });
```

Comment:
I like that you've done two tests here Ben - because however the test description are the same and they assert the same thing I think it is a little confusing. You could either
- put them in their own describe
- in the .then of the first - make your second request
- Look into a promise.all to do them concurrently
This isn't a requirement, rather a suggestion